src/components/ProfilePage/QuestCardDetailled/QuestCardDetailled.tsx

```diff
@@ -174,7 +189,7 @@ export const QuestCardDetailled = ({
                     lineHeight="18px"
                     color={'#ffffff'}
                   >
-                    {`${points}`}
+                    {`${formatDecimal(points)}`}
```

Comment:
we don't have decimals on points, is it necessary?

Reply:
We had points with many decimals which looked odd: This "formatDecimals" reduces decimals to 2. Wdyt?